hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -69,10 +69,24 @@ public class ContextTransactionProcessor implements Callable<CallOutcome> {
     private final TransactionProcessor processor;
     private final EvmActionTracer evmActionTracer;
     private final RootProxyWorldUpdater rootProxyWorldUpdater;
-    public final HevmTransactionFactory hevmTransactionFactory;
+    private final HevmTransactionFactory hevmTransactionFactory;
```

Comment:
it seems this pr does have at least one change that is not changing a comment?

Reply:
I must've forgotten this, I guess I just saw a possible minor change to make.